modules/openy_gc_auth/modules/openy_gc_auth_custom/src/Plugin/GCIdentityProvider/Custom.php

```diff
@@ -155,6 +163,7 @@ public function submitConfigurationForm(array &$form, FormStateInterface $form_s
       $this->configuration['email_verification_link_life_time'] = $form_state->getValue('email_verification_link_life_time');
       $this->configuration['email_verification_text'] = !empty($form_state->getValue('email_verification_text')) ? $form_state->getValue('email_verification_text')['value'] : '';
       $this->configuration['verification_message'] = !empty($form_state->getValue('verification_message')) ? $form_state->getValue('verification_message')['value'] : '';
+      $this->configuration['one_time_link_no_valid_message'] = !empty($form_state->getValue('one_time_link_no_valid_message')) ? $form_state->getValue('one_time_link_no_valid_message'): '';
```

Comment:
Please add `one_time_link_invalid_message` to `defaultConfiguration()`

Reply:
@hamrant 
Fixed. Please re-check it.